docs/airgap.md

```diff
@@ -0,0 +1,55 @@
+# Airgap support
+
+:::tip
+This page concerns only official XO Appliances, and not XO built from sources.
+:::
+
+XO has not been initially developped to support airgapped infrastructures but the following procedures have been tested and validated to permit its support.
+
+A separate XCP-ng pool with Internet access is necessary, it will be used as a temporary zone to register and upgrade the XOA.
```

Comment:
*and* feels weird to me here, but you're the native speaker :slightly_smiling_face: 

```suggestion
A separate XCP-ng pool with Internet access is necessary, and it will be used as a temporary zone to register and upgrade the XOA.
```